config/resources/document-domain.lisp

```diff
@@ -49,9 +49,6 @@
             (agendaitem                 :via ,(s-prefix "besluitvorming:geagendeerdStuk")
                                         :inverse t
                                         :as "agendaitem")
-            (agendaitem                 :via ,(s-prefix "ext:bevatReedsBezorgdAgendapuntDocumentversie")
-                                        :inverse t
-                                        :as "agendaitem")
```

Comment:
Deze relatie is de inverse relatie van de reeds gekoppelde documenten.
Als deze een probleem geeft bij reverse query zouden we dan deze beter hernoemen ipv weg te doen ?
```:as "linked-agendaitem"```  zoals lijn 48 (linked-subcase)

Reply:
De essentie hier is dat er op lijn `51` eenzelfde property key gebruikt wordt. Hernoemen zou kunnen, maar het feit dat we deze error voor het eerst tegenkomen impliceert dat deze keys nog niet gebruikt werden. Zolang ie niet gebruikt wordt, zou ik zeggen "weg ermee"?

Reply:
voor mij was het eerder een rechtzetting van 'deze relatie wordt niet gebruikt' of 'dit is een dubbele property'
De relatie wordt wel gebruikt, maar niet in de inverse richting zoals ze hier ligt.
Op deze plaats ligt de inverse relatie tussen documenten en reeds bezorgde documenten van agendapunt, daarom dat hernoemen naar linked-agendaitem correcter was dan verwijderen.

Deze inverse relatie hoeft hier niet expliciet te staan, tenzij we ooit uit de frontend willen querien:
"Geef me van dit document alle agendapunten die dit als "reeds gekoppeld document" hebben.
Deze specifieke query doen we nu niet.

Wegdoen of aanpassen heeft hetzelfde effect, zolang de dubbele "as agendaitem" maar weg is.
In eerste instantie vond ik hernoemen de propere oplossing.
Hangt eraf wat je verwacht van de lisp files en of we alle relaties in beide richtingen moeten leggen of niet.
Anders moeten we eens een opkuis van de lisp files doen en inverse relaties die niet zijn vermeld in het frontend model eruit gooien

Reply:
Klopt. Er valt idd ook wat op te kuisen. Ik denk dan meestal als ik iets tegen kom "weer eentje minder". Maar voor "afblijven tot we ineens alles doen", valt evengoed wat te zeggen. Zeg jij het maar. Ik zal 't doen.

Reply:
Approved, deze mag zo door dan